packages/router/src/lib/helpers/fulfill.ts

```diff
@@ -1,3 +1,5 @@
 import { signRouterFulfillTransactionPayload as _signRouterFulfillTransactionPayload } from "@connext/nxtp-utils";
 
 export const signRouterFulfillTransactionPayload = _signRouterFulfillTransactionPayload;
+
+export const WATCH_TOWER_BUFFER = 2 * 3600; // time buffer if watchtower can handle fulfill
```

Comment:
Let's call this `WATCHTOWER_CALL_TIMEOUT` ?

Reply:
Also, I *really* dislike this way of doing things. Whatever timeout we set here determines how long the window is for batching on the watchtower is... that's the same constant in two codebases.

A better idea: let's have the watchtower have an API to check whether a fulfill got collected into the latest batch.

Reply:
Opened an issue for this improvement https://github.com/connext/watchtower/issues/15